trio/tests/test_util.py

```diff
@@ -221,3 +223,106 @@ def test_module_metadata_is_fixed_up():
     # Also check methods
     assert trio.ssl.SSLStream.__init__.__module__ == "trio.ssl"
     assert trio.abc.Stream.send_all.__module__ == "trio.abc"
+
+
+# define a concrete class implementing the PathLike protocol
+# Since we want to have compatibility with Python 3.5 we need
+# to define the base class on runtime.
+BaseKlass = os.PathLike if hasattr(os, "PathLike") else object
```

Comment:
This is harmless, but I think it would work just as well if we left off the base class entirely and dropped the `issubclass` checks below.